setup.py

```diff
@@ -21,6 +21,8 @@
 extras["test"] = [
     "pytest",
     "pytest-xdist",
+    "tensorflow",
```

Comment:
Not following why we have to add TensoFlow here. Can't it be just `tensorboard`?